sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/parser/SqlDdlNodes.java

```diff
@@ -28,14 +27,6 @@
 public class SqlDdlNodes {
   private SqlDdlNodes() {}
 
-  /** Creates a CREATE TABLE. */
```

Comment:
This annoys me. I don't like this extra indirection in calcite but I also don't like deviating from calcite unnecessaraly. Not much you can do about that.